stable/datacube-dashboard/templates/ingress.yaml

```diff
@@ -17,26 +19,32 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
 spec:
-{{- if .Values.ingress.tls }}
-  tls:
-  {{- range .Values.ingress.tls }}
-    - hosts:
-      {{- range .hosts }}
-        - {{ . | quote }}
-      {{- end }}
-      secretName: {{ .secretName }}
-  {{- end }}
-{{- end }}
   rules:
+  {{- if $ingressRedirect -}}
   {{- range .Values.ingress.hosts }}
```

Comment:
LGTM. Only one suggestion - could you please indent this block of code for readability.